### ISSUE TITLE
Cut a new release

### ISSUE THREAD

**Issue by micahhausler** [linuxkit/linuxkit#3697]
**Description**

> v0.8 is old...
> @deitch in https://github.com/tinkerbell/hook/issues/63#issuecomment-863758111

Can we have a newish release cut? v0.8 is 11 months old

**Comment by micahhausler** [comment]
ping @deitch any update here?

**Comment by rn** [comment]
I agree we should have another release. Doing a release typically takes several days so it's a question of finding that time. I would also like to see the `runc/containerd` update to the latest but that is currently blocked on https://github.com/linuxkit/linuxkit/pull/3684 and ideally we also update alpine base https://github.com/linuxkit/linuxkit/pull/3702 before cutting a new relase

**Comment by biolauri** [comment]
Will this fix the current missing multi-arch builds in, e.g. `ca-certificates`? The `latest` tag is currently `linux/s390x` only (and older than some `v0.8` tags).

If yes, everything's fine. Otherwise, I'd open a new issue.

**Comment by NeilGerring** [comment]
Any update on getting a new release of linuxkit?

**Comment by jf** [comment]
> I agree we should have another release. Doing a release typically takes several days so it's a question of finding that time. I would also like to see the `runc/containerd` update to the latest but that is currently blocked on #3684 and ideally we also update alpine base #3702 before cutting a new relase

it seems like #3684 (*and* #3718, which supersedes it as per https://github.com/linuxkit/linuxkit/pull/3684#issuecomment-950129045) and #3702 are now fixed. Is there anything (new issues maybe?) holding us back from a new release?
